### ISSUE TITLE
window is not defined

### ISSUE THREAD

**Issue by maslianok** [mjackson/history#361]
Get this error when running tests
```
return window.navigator.userAgent.indexOf('Trident') === -1;
         ^

ReferenceError: window is not defined
```

version: history@3.2.0

**Comment by kwoninho-rainbow** [comment]
i had the same problem

**Comment by shripadk** [comment]
Fix for this issue: https://github.com/mjackson/history/pull/360

**Comment by jooj123** [comment]
I got the same with server side rendering, I moved my require that had window available - but the fix is the ideal way to go 👍

**Comment by mjackson** [comment]
Fixed in #360, released in v3.2.1.
